README.md

```diff
@@ -2,557 +2,87 @@
-{
-    id:2,
-    machine_type_id: 1,
-    user_id: 1,
-    name: Bright Tank 2,
-    cost: 5091,
-    active: TRUE,
-    created_at: 2019-07-16T21:52:00.823Z,
-    updated_at: 2019-07-16T21:52:00.823Z
-}
-```
-### POST Resources /api/v1/resources
-Request:
-```javascript
-{
-    resource_type_id: 1,
-    name: Bright Tank 3,
-    cost: 5091,
-    user_id: 1,
-    active: TRUE,
-    created_at: 213546f34,
-    updated_at: 235d64
-}
-```
-Response:
-```javascript
-{
-    id:2,
-    machine_type_id: 1,
-    user_id: 1,
-    name: Bright Tank 2,
-    cost: 5091,
-    active: TRUE,
-    created_at: 2019-07-16T21:52:00.823Z,
-    updated_at: 2019-07-16T21:52:00.823Z
-}
-```
-### PATCH Resources /api/v1/resources/:id
-Request:
-```javascript
-{
-    name: Bright Tank 4
-}
-```
-Response:
-```javascript
-{
-    id:3,
-    machine_type_id: 1,
-    user_id: 1,
-    name: Bright Tank 4,
-    cost: 5091,
-    active: TRUE,
-    created_at: 2019-07-16T21:52:00.823Z,
-    updated_at: 2019-07-16T21:52:00.823Z
-}
-```
-### GET Parts /api/v1/resources/:resource_id/parts
-Request:
-```javascript
-Eventually, there will be authorization in place via AWS and fetch calls.
-```
-Response:
-request: get /api/v1/resources/:resource_id/parts
-response:
-```json
-[
-    {
-        id: 1,
-        name: lever 2000,
-        inventory: 1,
-        active: true,
-        created_at: 2019-07-16T13:15:31.885Z,
-        updated_at: 2019-07-16T13:16:16.146Z
-    },
-    {
-        id: 2,
-        name: Tank patch,
-        inventory: 1,
-        active: true,
-        created_at: 2019-07-16T13:18:09.048Z,
-        updated_at: 2019-07-16T13:18:09.048Z
-    }
-]
-```
-### GET Parts /api/v1/resources/:resource_id/parts/:id
-Request:
-```javascript
-Eventually, there will be authorization in place via AWS and fetch calls.
-```
-Response:
-```javascript
-{
-    id: 2,
-    name: Tank patch,
-    inventory: 1,
-    active: true,
-    created_at: 2019-07-16T13:18:09.048Z,
-    updated_at: 2019-07-16T13:18:09.048Z
-}
-```
-### POST Parts /api/v1/resources/:resource_id/parts
-Request:
-```javascript
-{
-    name: Tank patch,
-    inventory: 1,
-}
-```
-Response:
-```javascript
-{
-    id: 2,
-    name: Tank patch,
-    inventory: 1,
-    active: true,
-    created_at: 2019-07-16T13:18:09.048Z,
-    updated_at: 2019-07-16T13:18:09.048Z
-}
-```
-### PATCH Parts /api/v1/resources/:resource_id/parts/:id
-Request:
-```javascript
-{
-    name: Tank lid
-}
-```
-Response:
-```javascript
-{
-    id: 2,
-    name: Tank lid,
-    inventory: 1,
-    active: true,
-    created_at: 2019-07-16T13:18:09.048Z,
-    updated_at: 2019-07-16T13:18:09.048Z
-}
-```
-### GET Tickets /api/v1/resources/:resource_id/tickets
-Request:
-```javascript
-Eventually, there will be authorization in place via AWS and fetch calls.
+Problem and Solution One
+Currently, Great Divide has a system that is unused and we sought to understand why and how we could design something that would be more likely to be utilized.
```

Comment:
Maybe it could read as:

Currently, Great Divide has an unused system and we sought to understand why and how we could design something that would be more likely to be utilized.